apps/email/js/cards/compose.js

```diff
@@ -361,6 +361,18 @@ return [
       };
     },
 
+    haveAddressesChanged: function() {
+      var addrs = this.extractAddresses();
+      function compareAddresses(a, b) {
+        return a.length == b.length && !a.some(function(e, i) {
+          return typeof b[i].address === 'undefined' || e !== b[i].address;
```

Comment:
> I do not understand what do you mean. Could you describe it in another way?

If it was better that we make sure the data is normalized to `{name: ..., address: ...}` (just without the name property in this case), then we could adapt the ` model.api.parseMailbox(addr).address` normalization to do that. However, since the other pathway from a compose/activity is working then we do not need to do that.